internal/integration/helpers.go

```diff
@@ -102,7 +109,17 @@ func testServer(tb testing.TB) (*serverenv.ServerEnv, *http.Client) {
 		tb.Fatal(err)
 	}
 
-	bs, err := storage.NewMemory(ctx)
```

Comment:
Go-style wise, it's probably better to keep this as is, then overwrite `bs` if the envvar exists:

```golang
bs, err := storage.NewMemory()
if err != nil {
  tb.Fatal(err)
}

if v := os.Getenv("GOOGLE_CLOUD_BUCKET"; v != "" && !testing.Short() {
  bs, err = storage.NewGoogleCloudStorage(ctx)
  if err != nil {
    tb.Fatal(err)
  }
}
```

Reply:
done